### PR TITLE
feat(driver): add schemaless raw insert for jdbc

### PR DESCRIPTION
**Pull request by huolibo**

### CODE REVIEW

File: source/client/src/clientJniConnector.c [hunk 1]
```diff
@@ -345,6 +345,51 @@ JNIEXPORT jlong JNICALL Java_com_taosdata_jdbc_TSDBJNIConnector_executeQueryImp(
...
+  (*env)->GetByteArrayRegion(env, jsql, 0, len, (jbyte *)str);
+  if ((*env)->ExceptionCheck(env)) {
+    // todo handle error
+  }
+
+  TAOS_RES *tres = taos_query_with_reqid(tscon, str, reqId);
+  int32_t   code = taos_errno(tres);
+
+  if (code != TSDB_CODE_SUCCESS) {
+    jniError("jobj:%p, conn:%p, code:%s, msg:%s", jobj, tscon, tstrerror(code), taos_errstr(tres));
```
Comment (sangshuduo) [inline source/client/src/clientJniConnector.c]: please print error code in hex format

File: source/client/src/clientJniConnector.c [hunk 2]
```diff
@@ -711,6 +756,44 @@ JNIEXPORT jlong JNICALL Java_com_taosdata_jdbc_TSDBJNIConnector_prepareStmtImp(J
...
+  if ((*env)->ExceptionCheck(env)) {
+    // todo handle error
+  }
+
+  TAOS_STMT *pStmt = taos_stmt_init_with_reqid(tscon, reqId);
+  int32_t    code = taos_stmt_prepare(pStmt, str, len);
+  taosMemoryFreeClear(str);
+  if (code != TSDB_CODE_SUCCESS) {
+    jniError("prepareStmtWithReqId jobj:%p, conn:%p, code:%s", jobj, tscon, tstrerror(code));
+    return JNI_TDENGINE_ERROR;
```
Comment (sangshuduo) [inline source/client/src/clientJniConnector.c:791]: seems the memory allocated for str is not freed

File: source/client/src/clientJniConnector.c [hunk 3]
```diff
@@ -1036,6 +1119,273 @@ JNIEXPORT jlong JNICALL Java_com_taosdata_jdbc_TSDBJNIConnector_schemalessInsert
...
+  for (int i = 0; i < numLines; ++i) {
+    jstring line = (jstring)((*env)->GetObjectArrayElement(env, lines, i));
+    (*env)->ReleaseStringUTFChars(env, line, c_lines[i]);
+  }
+
+  taosMemoryFreeClear(c_lines);
+
+  if (tres == NULL) {
+    jniError("execute schemaless insert with reqId result is NULL");
+    return JNI_OUT_OF_MEMORY;
```
Comment (sangshuduo) [inline source/client/src/clientJniConnector.c]: Interested why schemaless insert failed but return OUT OF MEMORY?

File: source/client/src/clientJniConnector.c [hunk 4]
```diff
@@ -1036,6 +1119,273 @@ JNIEXPORT jlong JNICALL Java_com_taosdata_jdbc_TSDBJNIConnector_schemalessInsert
...
+  for (int i = 0; i < numLines; ++i) {
+    jstring line = (jstring)((*env)->GetObjectArrayElement(env, lines, i));
+    (*env)->ReleaseStringUTFChars(env, line, c_lines[i]);
+  }
+
+  taosMemoryFreeClear(c_lines);
+
+  if (tres == NULL) {
+    jniError("execute schemaless insert with ttl result is NULL");
+    return JNI_OUT_OF_MEMORY;
```
Comment (sangshuduo) [inline source/client/src/clientJniConnector.c]: same question

File: source/client/src/clientJniConnector.c [hunk 5]
```diff
@@ -1036,6 +1119,273 @@ JNIEXPORT jlong JNICALL Java_com_taosdata_jdbc_TSDBJNIConnector_schemalessInsert
...
+  for (int i = 0; i < numLines; ++i) {
+    jstring line = (jstring)((*env)->GetObjectArrayElement(env, lines, i));
+    (*env)->ReleaseStringUTFChars(env, line, c_lines[i]);
+  }
+
+  taosMemoryFreeClear(c_lines);
+
+  if (tres == NULL) {
+    jniError("execute schemaless insert with TTL and reqId result is NULL");
+    return JNI_OUT_OF_MEMORY;
```
Comment (sangshuduo) [inline source/client/src/clientJniConnector.c]: ?

File: source/client/src/clientJniConnector.c (hunk 1 above)
Comment (huolibo) [reply source/client/src/clientJniConnector.c]: fixed!

File: source/client/src/clientJniConnector.c (hunk 3 above)
Comment (huolibo) [reply source/client/src/clientJniConnector.c]: this code has been removed！

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**